src/Compilers/CSharp/Test/Symbol/Compilation/CompilationAPITests.cs

```diff
@@ -2053,6 +2053,30 @@ public void PreviousSubmissionWithError()
             Assert.Throws<InvalidOperationException>(() => CreateSubmission("a + 1", previous: s0));
         }
 
+        [Fact]
+        public void CreateTupleTypeSymbol()
```

Comment:
@jaredpar, I don't mind annotating the tests (although I could not find `CompilerTrait` or `CompilerFeature`). But I don't much care about test counts as a measure.

One more thought on test organization, the current tests seem to be structured (parsing, symbol, semantic, emit). I don't mind moving away from that model, and towards one-test-file-per-new-feature, but it would be good to make it an explicit guideline.